### DISCUSSION TITLE
changfe colors

### DISCUSSION THREAD

**Discussion by ghost** [OpenRCT2/OpenRCT2#23145] [Q&A]
how to change the red font to another color?
i can't see it!

![afbeelding](https://github.com/user-attachments/assets/391ebf13-ca97-4a53-99f7-a1e90773948f)

**Comment by ZehMatt** [comment]
There is no option to change the color and will most likely never become one.

**Reply by ghost** [reply]
> There is no option to change the color and will most likely never become one.

then why is the most hard to read color on black background chosen? :-S

**Reply by ZehMatt** [reply]
It's the most common color used for errors in the console/terminal. Your screenshot seems perfectly readable, maybe it is rather your screen or some local color settings?
